src/python/ensembl/io/genomio/gff3/process.py

```diff
@@ -515,7 +518,7 @@ def _normalize_transcript_subfeatures(
                     f"Unrecognized exon type for {feat.type}: {feat.id}"
                     f" (for transcript {transcript.id} of type {transcript.type})"
                 )
-                print(message)
+                logging.warning(message)
```

Comment:
Same suggestion as before:
```suggestion
                logging.warning((
                    f"Unrecognized exon type for {feat.type}: {feat.id}"
                    f" (for transcript {transcript.id} of type {transcript.type})"
                ))
```